torch_xla/experimental/xla_sharding.py

```diff
@@ -355,16 +386,17 @@ def _get_group_assignment(
     # will be used to group replication devices.
     tile_dims = [d for d in partition_spec if d is not None]
 
-    group_list = [np.array(mesh.get_logical_mesh().tolist())]
+    logical_mesh = tile_assignment
```

Comment:
Not sure what this is about? Maybe we can stick on the same term either logical_mesh or tile_assignment.

Reply:
I've cleaned this up in the latest revision